shapeshift/web
# feat: opportunites chain filter

## Description



## Notice




## Pull Request Type


## Issue (if applicable)



closes #4087

## Risk

* add account modal
* dashboard/earn page



## Testing

* check we can still add accounts
* check we can filter opportunities page by chain, or all chains, and empty states are sane



### Engineering

as above



### Operations

as above



## Screenshots (if applicable)

by chain

<img width="703" alt="image" src="https://user-images.githubusercontent.com/88504456/229204963-bd921091-2743-4edd-a193-9e8381db9cd7.png">

mobile

<img width="699" alt="image" src="https://user-images.githubusercontent.com/88504456/229204895-166ac605-0d3d-4b44-aa4b-a6d182563577.png">


empty filter

<img width="886" alt="image" src="https://user-images.githubusercontent.com/88504456/229204796-78f1d8c6-e2ed-4141-b3e3-031a6a549b54.png">
